tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+            log.info(f"Filesystem got created from kubernetes Side")
+            k8s_validate = True
+        else:
+            log.error("Filesystem was not create at Kubernetes Side")
+            return False
+    if ceph_validate and k8s_validate:
+        return True
+    else:
+        return False
+
+
+def create_storageclass_rbd(kwargs):
+    """
+    Create a new CSI StorageClass
+    """
+    pool_name = kwargs.get('rbd_pool')
+    sc_name = kwargs.get('rbd_storageclass_name')
+    assert create_rbd_pool(pool_name=pool_name)
+    assert validate_pool_creation(pool_name)
+    if validate_pool_creation(pool_name):
+        log.info("Resource validation Passed")
+    else:
+        return 1
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_RBD_YAML, **kwargs
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new RBD StorageClass")
+    assert SC.create(yaml_file=TEMP_YAML_FILE)
+    assert validate_storageclass(sc_name)
+    return True
+
+
+def get_pool_get_sc_name():
+    """
+    This will generate random pool_name and storageclass name
+    :return:
+    """
+    sc_data = {}
+    pool_name = "pool-" + ''.join(
+        [random.choice(string.ascii_lowercase) for _ in range(4)])
+    sc_name = "ocsci-csi-" + pool_name
+    sc_data['rbd_storageclass_name'] = sc_name
+    sc_data['rbd_pool'] = pool_name
+    return sc_data
+
+
+def get_storageclass():
+    """
+    Funtion for getting all storageclass
+    :return: storageclass list
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+    ignore_sc = 'gp2'
+    for item in sample:
+        if item.metadata.name not in ignore_sc:
+            storageclass_list.append(item.metadata.name)
+    if len(storageclass_list) >= 1:
+        return True
+    else:
+        log.error("No StorageClass Found")
+        return False
+
+
+def create_secret(file_name):
+    """
+    This will create Secret file which will be used for creating StorageClass
+    :return:
+    """
+    secret_data = {}
+    user = "admin"
+    admin_byte = base64.b64encode(user.encode("utf-8"))
+    admin_base64 = str(admin_byte, "utf-8")
+    cmd = f"ceph auth get-key client.admin"
+    out = ocp.exec_ceph_cmd(cmd)
+    admin_key = out.get('key')
+    admin_key_byte = base64.b64encode(admin_key.encode("utf-8"))
+    admin_key_base64 = str(admin_key_byte, "utf-8")
+    secret_data['base64_encoded_admin_password'] = admin_key_base64
+    secret_data['base64_encoded_admin'] = admin_base64
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        file_name, **secret_data
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert SECRET.create(yaml_file=TEMP_YAML_FILE)
+    secret_list.append(file_y.get('metadata')['name'])
+    return True
+
+
+def create_pvc(**kwargs):
```

Comment:
you are only passing 1 argument here - count.
So no need for **kwargs.

It should be 
```python
def create_pvc(count=1)
```

And with that, you also taking care of line 350

Reply:
done